content/posts/calculus/integration_differentiation.md

```diff
@@ -0,0 +1,200 @@
++++
+title = '🔁 Integration and Differentiation: Inverse Operations'
+date = 2024-05-02T11:16:19+01:00
+tags = ["integral", "area under curve", "derivative", "tangent", "Riemann sum"]
+draft = false
++++
+
+<span style="background-color: lightgrey; border: 1px solid black; padding: 2px 10px; display: inline-flex; align-items: center;">
+  <details>
+    <summary><strong>Table of Contents</strong></summary>
+      {{< toc >}}
+  </details>
+</span>
+<br><br>
+
+Back in high school, I remember being introduced to the key concepts in calculus. First, the  <strong>derivative</strong> was presented as the slope of the tangent line to a curve at a given point. Then the  <strong>integral</strong> was defined as the area under the curve. And finally the leap of faith: both operations were inverses of each other. 
+
+I remember being confused about this statement we were asked to blindly accept. And college did not help much in that regard. We were given a bunch of rules to compute derivatives and integrals, but the connection between them was never made clear. And that is precisely what we are going to do in this post. We will start by revisiting the definitions of the <strong>derivative</strong> and the <strong>integral</strong>, and then we will try to gain some geometric intuition about why they reverse each other.
```

Comment:
It is easier to read this if written like: I remember being confused **by** this statement **that** we were asked to blindly accept.  ?